modules/c2c/murano2cloud.lua

```diff
@@ -14,36 +14,46 @@ local cloud2murano = require("c2c.cloud2murano")
 -- See all operations available in http://docs.exosite.com/reference/services/device2
 
 function murano2cloud.addIdentity(data)
-  local result = murano.services[murano2cloud.alias].addIdentity({ identity = data.identity })
-  if result and result.error then return result end
+  if "Dummycloudservice" ~= murano2cloud.alias then --if no actual cloud service, ignore
```

Comment:
Some of those operations are defined in the `Dummycloudservice` so the request can be seen on the requestbin page.

See https://github.com/exosite/getting-started-solution-template/blob/127e626d9a4e37c0676522bebeeb1595d00066f3/DummyCloudService.yaml#L171

That was an issue for the TTN integration?

Reply:
The actual issues were with listIdentities and getIdentityState - with those in there as they were, ExoSense will not enumerate devices and the specific functions fail with errors in the connector.

If there is a way to make it work with ExoSense with those intact, I'm all for it.

Reply:
ok, I ll check if I find a way.